registry/registry_test.go

```diff
@@ -893,6 +899,34 @@ func TestIsSecureIndex(t *testing.T) {
 	}
 }
 
+
+func TestTLSConfig(t *testing.T){
+	var index = makeIndex("/v1/")
```

Comment:
Index variable is used many times and always re declared, its better to  initialize it at the top then use it in all the tests.